playwright-e2e/tests/smoke-test.spec.ts

```diff
@@ -69,6 +71,12 @@ test("Smoke Test @smoke-test", async ({
   await addApplicationDocuments.uploadDocumentSmokeTest();
   await startApplication.addApplicationDocumentsInProgress();
 
+  await page.pause();
```

Comment:
await page.pause(); perhaps not needed?

Reply:
ah yes! good spot, probably need to start using the breakpoints instead..